File: core/src/main/java/cz/dynawest/jtexy/InvalidStateException.java

package cz.dynawest.jtexy;

/**
 *
 ** @author Ondrej Zizka
 */
public class InvalidStateException extends TexyException {

  public InvalidStateException() {
  }


  public InvalidStateException( String arg0 ) {
    super( arg0 );
  }


  public InvalidStateException( String arg0, Throwable arg1 ) {
    super( arg0, arg1 );
  }


  public InvalidStateException( Throwable arg0 ) {
    super( arg0 );
  }


}// class InvalidStateException

File: core/src/main/java/cz/dynawest/jtexy/InvalidStateException.kt
package cz.dynawest.jtexy

/**
 *
 * @author Ondrej Zizka
 */
class InvalidStateException : TexyException {
    constructor()
    constructor(arg0: String?) : super(arg0)
    constructor(arg0: String?, arg1: Throwable?) : super(arg0, arg1)
    constructor(arg0: Throwable?) : super(arg0)
}
